### ISSUE TITLE
[Adaptive Cards - Input Form]: Talkback is not announcing the role of the image while navigating on the 'Input Form' screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#490]
### Target Platforms

Android

### SDK Version

1.0.1

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: Android 16
Device: Samsung S24
Screen Reader: Talkback
Adaptive Card Android App version: 1.0.1

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select it.
7. Navigate all the controls under dropdown and select 'Input Form' card.
8. Navigate to the screen using left/right swipe gesture and observe the issue.

**Actual Result:**
While navigating the 'Input Form' screen using swipe gesture, Talkback is not announcing the role as 'image' for the 'Driver in great barrier reaf'. Currently it is announcing as **'Driver in great barrier reaf'.**

**Expected Result:**
Screen Reader should announce the role as 'Image' for the Great Barrier image. When the focus lands on it then it should announce as 'Driver in great barrier reaf image or graphic.'

**User Impact:**
It will impact the screen reader user as When TalkBack doesn’t announce the role or purpose of an image, users who are blind or visually impaired are left without crucial information. They won’t know if the image is decorative, informative, a button, or a link.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/89f891b6-d761-4638-a6a4-c459824f607c

### Card JSON

```JSON
NA
```
